README.adoc

```diff
@@ -3,21 +3,59 @@
 image:http://join.spinnaker.io/badge.svg[Slack Status,link=http://join.spinnaker.io]
 image:https://travis-ci.org/spinnaker/spinnaker.svg?branch=master["Build Status", link="https://travis-ci.org/spinnaker/spinnaker"]
 
-Spinnaker is a continuous delivery platform for releasing changes with high velocity and confidence.
+= Welcome to the Spinnaker Project
 
-This repository is used only to store and organize ancillary configuration, build, and testing tools for the Spinnaker project. The core code making up Spinnaker's microservices is found in the other https://github.com/spinnaker[Spinnaker repositories].
+Spinnaker is an open-source continuous delivery platform for releasing software changes with high velocity and confidence.
+Through a powerful abstraction layer, Spinnaker provides compelling tooling that empowers developers to own their application code from commit to delivery.
+
+Inside Spinnaker, using a GUI (graphical user interface), or as code, manage deployments and cloud resources such as: 
+
+- VM "bake" deployments to a cloud
+- Container deployments to a cloud
+- Container deployments to Kubernetes
+- Load balancers
+- Security groups
+- Server groups
+- Clusters
+- Firewalls
+- Functions
+
+
+Facilitate GitOps, and grant developers control of provisioning infrastructure for apps. Natively, use Spinnaker’s cloud driver to deploy to all of the major cloud providers, and the Kubernetes V2 provider for Kubernetes deployments. You may even orchestrate configuration and firmware changes as part of Spinnaker delivery pipelines.
+
+= Why Do I Need Spinnaker?
+
+With Spinnaker, create a “paved road” for application delivery, with guardrails that ensure only valid infrastructure and configuration reach production.
+Free development teams from burdensome ops provisioning while automating reinforcement of business and regulatory requirements. Delivery automation
+strategies such as canary deployments provide the safety necessary to capture value from quick innovation, while protecting against business and end-user
+ impact. As the most mature and widely productionalized continuous delivery platform, Spinnaker can apply the expertise of Netflix, Google, Microsoft,
+ and Amazon to your SDLC. User companies include Target, Salesforce, Airbnb, Cerner, Adobe, and JPMorgan Chase.
+ 
+= Tech Specs
```

Comment:
TIL about the `= ` prefix making a header and horizonal line - thanks for the tip!